munit-scalacheck/shared/src/main/scala/munit/ScalaCheckSuite.scala

```diff
@@ -49,11 +49,14 @@ trait ScalaCheckSuite extends FunSuite {
 
   private def propToTry(prop: Prop, test: Test): Try[Unit] = {
     import ScalaCheckTest._
-    val seed =
+    def seed =
       scalaCheckTestParameters.initialSeed.getOrElse(
         Seed.fromBase64(scalaCheckInitialSeed).get
       )
-    val result = check(scalaCheckTestParameters, prop.useSeed(test.name, seed))
+    val result = check(
+      scalaCheckTestParameters,
+      Prop(genParams => prop(genParams.withInitialSeed(seed)))
+    )
```

Comment:
this way if the user hasn't overridden the default seed, we recompute it in the `Prop` lambda (which is what `viewSeed` does in vanilla ScalaCheck)